src/js/editor/editor.js

```diff
@@ -293,7 +306,7 @@ class Editor {
     this.run(postEditor => {
       let nextPosition = postEditor.deleteAtPosition(position, direction, {unit});
       postEditor.setRange(nextPosition);
-    });
+    }, EDITOR_RUN_LAST_ACTION_STATE.DELETE);
```

Comment:
@disordinary Sorry to let this linger — we were at EmberFest.
Yeah, hiding the second arg with a private `_run` totally makes sense to me. 👍